internal/mcp_server.go

```diff
@@ -374,13 +386,22 @@ func (s mcpServer) ApplyTool(ctx context.Context, req mcp.CallToolRequest) (*mcp
 	return mcp.NewToolResultText("The objects were successfully applied."), nil
 }
 
-func (s mcpServer) ReplayTool(ctx context.Context, req mcp.CallToolRequest) (*mcp.CallToolResult, error) {
-	sloName := req.Params.Arguments["slo"].(string)
-	projectName := req.Params.Arguments["project"].(string)
-	fromStr := req.Params.Arguments["from"].(string)
-
+func (s *mcpServer) ReplayTool(
+	ctx context.Context,
+	_ mcp.CallToolRequest,
+	args mcpToolArguments,
+) (*mcp.CallToolResult, error) {
+	if args.From == "" {
+		return nil, fmt.Errorf("'from' argument is required")
```

Comment:
This should be checked in the tool definition by `mcp.Required()` option, not in the implementation